source/includes/mongodump-restore-document-order.rst

```diff
@@ -0,0 +1,2 @@
+``mongodump`` and ``mongorestore`` preserve document order during the
```

Comment:
No, you need to set: https://www.mongodb.com/docs/database-tools/mongorestore/#std-option-mongorestore.--maintainInsertionOrder